spec/http/1.0/openapi.json

```diff
@@ -298,13 +291,7 @@
             "additionalProperties": true
           }
         },
-        "required": [
-          "id",
-          "name",
-          "description",
-          "input_schema",
-          "output_schema"
```

Comment:
Removed input_ and output_ from `required` to reflect the fact that these MAY be null.

Reply:
IMO these should be required.

* The input should be explicitly an empty object to denote a parameterless function. You could argue in this case a bit if you really believe that positional arguments should be supported.
* The output should be very explicit: Either it is known and can be specifically denoted as null or some other value, or it is not known and should be marked as any.